jekyll/_posts/2018-10-02-create-a-simple-macro-part-2.md

```diff
@@ -0,0 +1,173 @@
+
+---
+title: "Creating a simple macro 2nd part"
+author: Antonis Geralis
+---
+
+<div class="sidebarblock">
+  <div class="content">
+    <div class="title">Guest post</div>
+    <div class="paragraph">
+      This is a guest post by Antonis Geralis. If you would like to publish 
+      articles as a guest author on nim-lang.org then get in touch with us via
+      <a href="https://twitter.com/nim_lang">Twitter</a> or
+      <a href="https://nim-lang.org/community.html">otherwise</a>.
+    </div>
+  </div>
+</div>
+
+## Creating a macro for a state machine
+
+This time we will be building a state machine. Macros can be useful here in
```

Comment:
You need to elaborate a bit what "state machine" means here, I'm afraid. (Yes, I personally do know what a state machine is.)